From 92f6d4db0b61e59359ca0ccbbea5354b17c39362 Mon Sep 17 00:00:00 2001
From: blunden <blunden2@gmail.com>
Date: Fri, 14 Dec 2012 03:59:13 +0100
Subject: [PATCH 06/18] LatinIME: Enable spellchecker for additional languages

We include more dictionaries than stock builds so let's
also enable spell checking for languages that use only
the supported unicode characters.

Change-Id: Iff9aea56edde10e47a754df39682f4f72c269756

LatinIME: Enable Greek spell checking

* We already ship a dictionary for Greek

Change-Id: I99ee7c98a88515c4c69b654210c10696135d7560
---
 java/res/xml/spellchecker.xml | 20 ++++++++++++++++++++
 1 file changed, 20 insertions(+)

diff --git a/java/res/xml/spellchecker.xml b/java/res/xml/spellchecker.xml
index d2da17f9d..c9973d204 100644
--- a/java/res/xml/spellchecker.xml
+++ b/java/res/xml/spellchecker.xml
@@ -63,4 +63,24 @@
             android:label="@string/subtype_generic"
             android:subtypeLocale="hu"
     />
+    <subtype
+            android:label="@string/subtype_generic"
+            android:subtypeLocale="da"
+    />
+    <subtype
+            android:label="@string/subtype_generic"
+            android:subtypeLocale="fi"
+    />
+    <subtype
+            android:label="@string/subtype_generic"
+            android:subtypeLocale="nb"
+    />
+    <subtype
+            android:label="@string/subtype_generic"
+            android:subtypeLocale="sv"
+    />
+    <subtype
+            android:label="@string/subtype_generic"
+            android:subtypeLocale="el"
+    />
 </spell-checker>
-- 
2.17.1

